flink-connectors/flink-sql-connector-aws-kinesis-streams/pom.xml

```diff
@@ -36,6 +36,12 @@
 			<groupId>org.apache.flink</groupId>
 			<artifactId>flink-connector-aws-kinesis-streams</artifactId>
 			<version>${project.version}</version>
+			<exclusions>
+				<exclusion>
+					<groupId>org.apache.flink</groupId>
+					<artifactId>flink-connector-base</artifactId>
```

Comment:
should be unncessary because the only remaining transitive Flink dependency after shading is flink-table-common